### DISCUSSION TITLE
VS code LSP trace functionality

### DISCUSSION THREAD

**Discussion by ro-gun** [Shopify/ruby-lsp#2717] [Q&A] [answered]
After cloning the repo locally, the vs code setting shows unknown configuration setting.

![image](https://github.com/user-attachments/assets/6c9c835e-9a57-4353-b3ba-33acf98a8a7a)

Is there a way to solve it?

**Comment by vinistock** [comment, marked as answer]
Yeah, the reason that shows up is because we are not declaring the tracing setting in our package.json. It's used by the LSP client package anyway, so everything works, but it's definitely weird for it to say unknown.

I think it makes sense to declare it [here](https://github.com/Shopify/ruby-lsp/blob/c82ec131536cc443fd236051de5c5c428870cdfb/vscode/package.json#L158).
